.rubocop.yml

```diff
@@ -1 +1,35 @@
 inherit_from: .rubocop_todo.yml
+
+# Cop supports --auto-correct.
+# Configuration parameters: AllowForAlignment, ForceEqualSignAlignment.
+Style/ExtraSpacing:
+  Enabled: true
+
+# Cop supports --auto-correct.
+# Configuration parameters: IndentationWidth.
+Style/IndentAssignment:
+  Enabled: true
```

Comment:
I would like to get to non-inlined versions eventually, I don't like much the fact that when you change one assignment, the whole block needs to get updated